ervinu/web content/style.css

```diff
@@ -0,0 +1,105 @@
+body {
+    background-color: rgb(211, 207, 207);
+    margin: 10px 5px;
+    font-family: sans-serif;
+}
+
+#top-container {
+    background-color: #ffffff;
+    margin: 5px;
+}
+
+#title h2, #title a {
+    display: inline-block;
+    margin: 5px 10px;
+}
+
+a {
+text-decoration: none;
+}
+
+.carousel {
+    background-color: rgb(255, 255, 255);
+    height: 330px;
+    width: 99%;
+    position: relative;
+    display: flex;
+}
+
+.carousel p {
+    margin: 4px 0px;
```

Comment:
Sometime you put 5px, other time 4px. Try to use this:

`0px, 2px, 4px, 8px, 16px, 32px, 64px` this way the site will be uniform